stl/inc/atomic

```diff
@@ -1177,10 +1195,17 @@ struct _Atomic_integral_facade : _Atomic_integral<_Ty> {
     // We make the primary functions non-volatile for better debug codegen, as non-volatile atomics
     // are far more common than volatile ones.
     using _Base::fetch_add;
+

```

Comment:
This newline appears to be inconsistent with the functions below.

Reply:
Yepp